modules/KalturaSupport/components/share/share.js

```diff
@@ -366,8 +401,12 @@
 			// close button override
 			$(".share .icon-close").on("mousedown", function(e){
 				_this.closeScreen();
+			})
+			.keyup(function (e) {
+				if(e.keyCode === 13){
+					_this.closeScreen();
```

Comment:
there are keybindings helpers in player

Reply:
@MaksymKorniiov  can you please review this request?

Reply:
@OrenMe we looked in the shortcut plugin and did not really understand what you meant here. 
We added here the option to close the share screen with keyboard. As you see in share screen http://player.kaltura.com/docs/ you may focus with KB on the close button but it is not triggerable with space nor Enter. This specific key-bind fixes that.

Reply:
What I meant is that we have a keybinding helper for general keybindings.
the keybindings are cross components. e.g. are registered on top level of player, and I see this is only for screen component.
We don't have any instrumentation to register globally keyboard events but assign them to specific target, so unfortunately my request is currently not relevant unless we refactor.
I'm removing my comment then.